### PR TITLE
Make sure duplicate event instances are not created

### LINKED ISSUE #1732: ensure that manual update of youtube link does not create new eventinstance
seems like updating youtube link manually causes a new event instance to be created?  would be great to avoid that ...

### PR DESCRIPTION
**Pull request by lokeshh**
Fixes #1732

### CODE REVIEW

File: app/views/events/show.html.erb [hunk 1]
```diff
@@ -98,7 +98,7 @@
 
 <% if user_signed_in? %>
 
-    <% hangout_is_live = @hangout.try!(:live?) %>
+    <% hangout_is_live = @recent_hangout.try!(:live?) %>
```
Comment (lokeshh) [inline app/views/events/show.html.erb]: The bug is caused by `@hangout` being always nil because its nowhere initialized. My guess is that it is a mistake by author which rather meant to write `@recent_hangout` and not `@hangout`.

Comment (tansaku) [inline app/views/events/show.html.erb]: interesting

File: features/step_definitions/hangout_steps.rb [hunk 2]
```diff
@@ -182,4 +185,7 @@
   page.should have_field('hangout_url', with: @hangout_url)
 end
 
-
+Then(/^a separate event instance is not created$/) do
+  expect(EventInstance.where("created_at >= ?", Time.now.beginning_of_day).size)
```
Comment (tansaku) [inline features/step_definitions/hangout_steps.rb]: I worry slightly that this test is dependent on the current time - if the test suite runs over the midnight boundary it could pass for the wrong reason.  Could we hard code the date based on the date set at the beginning of the cuke test?

Comment (lokeshh) [inline features/step_definitions/hangout_steps.rb]: I think the date coded at the beginning of the cuke test will persist to this step. Let me check using byebug.

Comment (lokeshh) [inline features/step_definitions/hangout_steps.rb]: I just checked, yes the date set in first step persist, so it doesn't matter when the test is ran. Time.now would return 2014-02-07 07:01:41 +0000

Comment (tansaku) [inline features/step_definitions/hangout_steps.rb]: yeah, I guess you're getting beginning of day - which is almost constant there, unless the test took a day to run, and that's a silly corner case, so I guess we're okay, but in general you should avoid using Time like that in a test

Comment (lokeshh) [inline features/step_definitions/hangout_steps.rb]: ok
